bases/br_inep_indicadores_educacionais/curso_de_para_anual/table_config.yaml

```diff
@@ -0,0 +1,287 @@
+# inicializando este arquivo de configuração, você pode apontar a função para uma amostra de dados que
+# preencherá automaticamente as colunas.
+# Algumas colunas existirão apenas na tabela final, você as construirá em `publish.sql`.
+# Para esses, defina is_in_staging como False.
+# Além disso, você deve adicionar as colunas de partição aqui e definir is_partition como True.
+columns: # REQUIRED
+  
+    -   
+        name: id_ies
+        description: Código de identificação da instituição de educação superior
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: nome_ies
+        description: Nome de identificação da instituição de educação superior
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: tipo_categoria_administrativa
+        description: Código da categoria Administrativa da IES no último ano de análise
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: tipo_organizacao
+        description: Código da organização acadêmica no último ano de análise.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_curso
+        description: Código de identificação do curso gerado pelo E-MEC
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: nome_curso
+        description: Nome de identificação do curso gerado pelo E-MEC
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_regiao
+        description: Código da Região Geográfica do local de oferta do curso gerado pelo E-MEC no último ano de análise.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_uf
+        description: Código da Unidade da Federação do local de oferta do curso gerado pelo E-MEC no último ano de análise.
+        is_in_staging: True # Bool [True, False], whether the column is in the staging table
+        is_partition: False # Bool [True, False], whether the column is a partition.
+  
+    -   
+        name: id_municipio
```

Comment:
Se essa coluna for o ID Município - IBGE 7 Dígitos que tem no `basedosdados.br_bd_diretorios_brasil.municipios`, então pode remover as colunas `id_uf` e `id_regiao`, pois conseguimos mapear essas informações só com o `id_municipio`. Caso não seja, ai tem que mudar no nome da coluna, pq `id_municipio` está padronizado como ID Município - IBGE 7 Dígitos.